src/client/state/actions/sagaActions.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable */
```

Comment:
I don't seem to understand having a separate bunch of actions for sagas.. could we not just use regular actions?